server/src/dtos/time-bucket.dto.ts

```diff
@@ -5,72 +5,202 @@ import { AssetOrder, AssetVisibility } from 'src/enum';
 import { Optional, ValidateAssetVisibility, ValidateBoolean, ValidateUUID } from 'src/validation';
 
 export class TimeBucketDto {
+  @ApiProperty({
+    type: 'string',
+    format: 'uuid',
+    required: false,
+    description: 'Filter assets by specific user ID',
+  })
```

Comment:
It's a bit annoying that you repeat a lot of stuff here that's already in the other decorators. I thought you can just add a comment above the field and it'll automatically turn it into a description? So like

```ts
// Filter assets by specific user Id
@ValidateUUID({ optional: true })
userId?: string;
```